aspnetcore/fundamentals/static-files.md

```diff
@@ -1,223 +1,253 @@
+> [!IMPORTANT]
+> `UseDefaultFiles` must be called before `UseStaticFiles` to serve the default file. `UseDefaultFiles` is a URL rewriter that doesn't actually serve the file. Enable the static file middleware via `UseStaticFiles` to serve the file.
 
-With `UseDefaultFiles`, requests to a folder will search for:
+With `UseDefaultFiles`, requests to a folder search for:
 
-* default.htm
-* default.html
-* index.htm
-* index.html
+* *default.htm*
+* *default.html*
+* *index.htm*
+* *index.html*
 
-The first file found from the list will be served as if the request was the fully qualified URI (although the browser URL will continue to show the URI requested).
+The first file found from the list is served as though the request were the fully qualified URI. The browser URL continues to reflect the URI requested.
 
-The following code shows how to change the default file name to *mydefault.html*.
+The following code changes the default file name to *mydefault.html*:
 
-[!code-csharp[Main](static-files/sample/StartupDefault.cs?name=snippet1)]
+[!code-csharp[](static-files/samples/1x/StartupDefault.cs?name=snippet_ConfigureMethod)]
 
 ## UseFileServer
 
-`UseFileServer` combines the functionality of `UseStaticFiles`, `UseDefaultFiles`, and `UseDirectoryBrowser`.
+[UseFileServer](/dotnet/api/microsoft.aspnetcore.builder.fileserverextensions.usefileserver#Microsoft_AspNetCore_Builder_FileServerExtensions_UseFileServer_Microsoft_AspNetCore_Builder_IApplicationBuilder_) combines the functionality of `UseStaticFiles`, `UseDefaultFiles`, and `UseDirectoryBrowser`.
 
-The following code enables static files and the default file to be served, but does not allow directory browsing:
+The following code enables the serving of static files and the default file. Directory browsing isn't enabled.
 
 ```csharp
 app.UseFileServer();
-   ```
+```
 
-The following code enables static files, default files and  directory browsing:
+The following code builds upon the parameterless overload by enabling directory browsing:
 
 ```csharp
 app.UseFileServer(enableDirectoryBrowsing: true);
-   ```
+```
 
-See [Considerations](#considerations) on the security risks when enabling browsing. As with `UseStaticFiles`, `UseDefaultFiles`, and `UseDirectoryBrowser`, if you wish to serve files that exist outside the `web root`, you instantiate and configure an `FileServerOptions` object that you pass as a parameter to `UseFileServer`. For example, given the following directory hierarchy in your Web app:
+See [Considerations](#considerations) on the security risks when enabling browsing. As with `UseStaticFiles`, `UseDefaultFiles`, and `UseDirectoryBrowser`, if you wish to serve files that exist outside the web root, instantiate and configure an `FileServerOptions` object that you pass as a parameter to `UseFileServer`. For example, consider the following directory hierarchy in your web app:
```

Comment:
>As with `UseStaticFiles`, `UseDefaultFiles`, and `UseDirectoryBrowser`, if you wish to serve files that exist outside the web root, instantiate and configure an `FileServerOptions` object that you pass as a parameter to `UseFileServer`.

Too long and too much steve code speak. If you wish is too colloquial.  Do we need to repeat?  As with `UseStaticFiles`, `UseDefaultFiles`, and `UseDirectoryBrowser`.  How about something like: The follow code serves files outside web root: